string/string.c

```diff
@@ -86,6 +86,9 @@ int strncasecmp(const char *s1, const char *s2, size_t n)
 			return 1;
 	}
 
+	if (k < n && (tolower(*p) != tolower(*(s2 + k))))
+		return -1;
```

Comment:
New style is advised for new code
```suggestion
	if (k < n && (tolower(*p) != tolower(*(s2 + k)))) {
		return -1;
	}
```